### PR TITLE
Adding Conditional Access Policy Implementation Instructions to AAD 1.1

### LINKED ISSUE #982: Enhance AAD policy 1.1 instructions by including a conditional access policy template
# 💡 Summary #

Improve the implementation instructions for AAD policy 1.1 by adding a conditional access policy template that shows the specific configuration that Scuba expects, similar to how we address all the other policies related to conditional access. See user feedback received from https://github.com/cisagov/ScubaGear/issues/941#issuecomment-1973916754

Find the instruction that links to Microsoft in the screenshot below and replace it with a template that looks similar to the instructions for policy 2.2. It should provide the exact configuration that Scuba expects.

image

Replace it with a template that looks similar to the screenshot below but describes how to disable legacy authentication instead:

image

## Motivation and context ##

This makes the documentation easier for users to follow, especially when they run ScubaGear and may not know why their assessment is failing.

### PR DESCRIPTION
**Pull request by dagarwal-mitre**
#  #
  

## 🗣 Description ##

Updated the implementation guide to follow template of other CAP making it easier on user

## 💭 Motivation and context ##

closes #982 

## 🧪 Testing ##

Read the updated verbose, and ensure it is clear and accurate directions

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] *All* future TODOs are captured in issues, which are referenced in code comments.
- [x] These code changes follow the ScubaGear [content style guide](https://github.com/cisagov/ScubaGear/blob/main/CONTENTSTYLEGUIDE.md).
- [x] Related issues these changes resolve are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] Unit tests added/updated to cover PowerShell and Rego changes.
- [x] Functional tests added/updated to cover PowerShell and Rego changes.
- [x] All relevant functional tests passed.
- [x] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [x] PR passed smoke test check.
- [x] Feature branch has been rebased against changes from parent branch, as needed

  Use `Rebase branch` button below or use [this](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request) reference to rebase from the command line.
- [x] Resolved all merge conflicts on branch
- [x] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

File: PowerShell/ScubaGear/baselines/aad.md [hunk 1]
```diff
@@ -88,9 +88,19 @@ Legacy authentication SHALL be blocked.
...
 
-- Create a [Conditional Access policy to block legacy authentication](https://learn.microsoft.com/en-us/entra/identity/conditional-access/howto-conditional-access-policy-block-legacy).
+2. Create a Conditional Access policy to block legacy authentication
+
+<pre>
+  Users > Include > <b>All users</b>
+
+  Target resources > Cloud apps >  Include > <b>All cloud apps</b>
+
+  Conditions > Client apps > Configure > Yes > Legacy authentication clients > Select only <b>Exchange ActiveSync clients</b> and <b>Other clients</b>
```
Comment (tkol2022) [inline PowerShell/ScubaGear/baselines/aad.md:100]:
```suggestion
  Conditions > Client apps > Configure > <b>Yes</b> > Legacy authentication clients > Select only <b>Exchange ActiveSync clients</b> and <b>Other clients</b>
```

**Review by tkol2022 — CHANGES_REQUESTED** [review]
Nice work. I reviewed the hyperlink in step 1 and it works fine. For step 2 I verified that the instruction template is accurate and I added some suggested bold to one of the words that you can simply commit if you agree with it.

**Review by ahuynhMITRE — APPROVED** [review]
no additional comments to this update.

**Comment by dagarwal-mitre** [comment]
@nanda-katikaneni this PR is ready to merge

**Review by tkol2022 — APPROVED** [review]

**Comment by tkol2022** [comment]
@nanda-katikaneni I believe this is ready to go.

**Pull request merged**